dz7/task34.py

```diff
@@ -0,0 +1,34 @@
+# Задача 34:  Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. Поскольку разобраться в его
+# кричалках не настолько просто, насколько легко он их придумывает, Вам стоит написать программу.
+# Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения 
+# одинаковое. Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами.
+# Фразы отделяются друг от друга пробелами. Стихотворение  Винни-Пух вбивает в программу с клавиатуры. 
+# В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+
+# *Пример:*
+
+# **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да    
+# **Вывод:** Парам пам-пам  
+
+poem = list(input('Введите считалку Винни через пробел:  ').split())
+print(poem)
+
+def find_rhythm(poem):
+    vowels_letters = ['у','е','ы','а','о','э','я','и','ю']
+    count = 0
+    parity = True
+    for phrase in poem:
+        for letter in phrase:
+            if letter in vowels_letters:
+                count += 1
```

Comment:
Смотрите какой вариант решения я увидел у одного студента. Это просто гениально.
```suggestion
            count += letter in vowels_letters
```
Подумайте почему?